src/dbnode/storage/index/block.go

```diff
@@ -799,7 +813,8 @@ func (b *block) Query(
 	}
 
 	size := results.Size()
-	batch := b.docsPool.Get()
+	docsPool := b.opts.DocumentArrayPool()
```

Comment:
Super nit: `var` block would be nice here.

Reply:
Sure yeah.